test/e2e/framework/authorizer_util.go

```diff
@@ -46,6 +47,12 @@ func WaitForAuthorizationUpdate(c v1beta1authorization.SubjectAccessReviewsGette
 	}
 	err := wait.Poll(policyCachePollInterval, policyCachePollTimeout, func() (bool, error) {
 		response, err := c.SubjectAccessReviews().Create(review)
+		// GKE doesn't enable the SAR endpoint.  Without this endpoint, we cannot determine if the policy engine
+		// has adjusted as expected.  In this case, simply wait one second and hope it's up to date
+		if apierrors.IsNotFound(err) {
```

Comment:
> would be nice to log this is happening

If I end up red.  Otherwise I'd rather not give up green on it since this unblocks the queue.